internal/kafka/internal/services/kafka_instance_types.go

```diff
@@ -0,0 +1,57 @@
+package services
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+)
+
+//go:generate moq -out kafka_instance_types_moq.go . SupportedKafkaInstanceTypesService
+type SupportedKafkaInstanceTypesService interface {
+	GetSupportedKafkaInstanceTypesByRegion(providerId string, regionId string) ([]config.KafkaInstanceType, *errors.ServiceError)
+}
+
+type supportedKafkaInstanceTypesService struct {
+	providerConfig *config.ProviderConfig
+	kafkaConfig    *config.KafkaConfig
+}
+
+func NewSupportedKafkaInstanceTypesService(providerConfig *config.ProviderConfig, kafkaConfig *config.KafkaConfig) SupportedKafkaInstanceTypesService {
+	return &supportedKafkaInstanceTypesService{
+		providerConfig: providerConfig,
+		kafkaConfig:    kafkaConfig,
+	}
+}
+
+func (t supportedKafkaInstanceTypesService) GetSupportedKafkaInstanceTypesByRegion(providerId string, regionId string) ([]config.KafkaInstanceType, *errors.ServiceError) {
```

Comment:
```suggestion
func (t *supportedKafkaInstanceTypesService) GetSupportedKafkaInstanceTypesByRegion(providerId string, regionId string) ([]config.KafkaInstanceType, *errors.ServiceError) {
```

See previous comment.